daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -292,11 +300,27 @@ func (l *logStream) BufSize() int {
 
 // Log submits messages for logging by an instance of the awslogs logging driver
 func (l *logStream) Log(msg *logger.Message) error {
+	// In the blocking case we have already called create in New()
+	if l.logNonBlocking {
+		err := l.create()
```

Comment:
nit: combine as `if err := l.create(); err != nil {`

Reply:
Good catch, I changed this.